src/sentry_plugins/github/webhooks/events/push.py

```diff
@@ -78,25 +86,24 @@ def _handle(self, event, organization_id, is_apps):
                                 # even if we can't find a user, set to none so we
                                 # don't re-query
                                 gh_username_cache[gh_username] = None
-                                try:
-                                    user = User.objects.filter(
-                                        social_auth__provider="github",
-                                        social_auth__uid=gh_user["id"],
-                                        org_memberships=organization_id,
-                                    )[0]
-                                except IndexError:
-                                    pass
-                                else:
-                                    author_email = user.email
-                                    gh_username_cache[gh_username] = author_email
-                                    if commit_author is not None:
-                                        try:
-                                            with transaction.atomic():
-                                                commit_author.update(
-                                                    email=author_email, external_id=external_id
-                                                )
-                                        except IntegrityError:
-                                            pass
+                                for social_auth in UserSocialAuth.objects.filter(
+                                    provider="github", uid=gh_user["id"]
+                                ):
+                                    member = OrganizationMember.objects.filter(
+                                        organization_id=organization_id, user_id=social_auth.user.id
+                                    ).first()
```

Comment:
This will be ideally the org member mapping table in the future. This is a workaround to be able to drop `Organization.members` column.